### DISCUSSION TITLE
Don't see link balance

### DISCUSSION THREAD

**Discussion by KiseSaiyajin** [Cyfrin/foundry-full-course-cu#3029] [Q&A] [answered]
Maybe it sounds very stupid, but I don't see my link balance, but in history I see my balance correctly. 
![image](https://github.com/user-attachments/assets/22486ad6-9d3e-469e-ab19-a8d7d3e20f1b)
![image](https://github.com/user-attachments/assets/1b98bd59-49ce-4f40-be04-e43be46112c5)
Why my link balance didn't change? Maybe something wrong?

**Comment by KMean** [comment, marked as answer]
I think it's a UI bug, to me happen just on sepolia. Can you try on Amoy and see what happen?

**Reply by KiseSaiyajin** [reply]
It is really UI bug, I tried to fund using website, same issue. Balance is still zero.

**Comment by EngrPips** [comment]
It's tough to say what is wrong, but seems like a problem on the end of Chainlink.
